src/panels/call-to-action/module.js

```diff
@@ -66,26 +67,36 @@ class CallToActionCtrl extends PanelCtrl {
 
   getOrgDetails() {
     var self = this;
-    var p = this.backendSrv.get('api/plugin-proxy/raintank-worldping-app/api/grafana-net/profile/org');
-    p.then((resp) => {
-      self.org = resp;
-
-      const millionChecksPerMonth = Math.ceil(parseInt(self.org.checksPerMonth, 10) / 100000) / 10;
-      if (millionChecksPerMonth > 1000) {
-        self.org.strChecksPerMonth = 'using ' + Math.ceil(millionChecksPerMonth / 1000) + ' Billion checks/mo';
-      } else if (millionChecksPerMonth > 0) {
-        self.org.strChecksPerMonth = 'using ' + millionChecksPerMonth + ' Million checks/mo';
-      } else {
-        self.org.strChecksPerMonth = 'not using any checks yet';
-      }
-
-      self.requiresUpgrade = self._requiresUpgrade();
-      self.currentlyTrial = self._currentlyTrial();
-      self.aboveFreeTier = self._aboveFreeTier();
-    }, (resp) => {
-      self.alertSrv.set("failed to get Org Details", resp.statusText, 'error', 10000);
-    });
-    return p;
+    if(self.dashboard.snapshot){
+      return this.onOrgDetails(self.panel.snapshotData.org);
+    } else {
+      var p = this.backendSrv.get('api/plugin-proxy/raintank-worldping-app/api/grafana-net/profile/org');
+      p.then((resp) => {
+        self.panel.snapshotData.org = resp;
+        self.onOrgDetails(resp);
+      }, (resp) => {
+        self.alertSrv.set("failed to get Org Details", resp.statusText, 'error', 10000);
+      });
+      return p;
+    }
+  }
+
+  onOrgDetails(resp) {
+    var self = this;
+    self.org = resp;
+
+    const millionChecksPerMonth = Math.ceil(parseInt(self.org.checksPerMonth, 10) / 100000) / 10;
+    if (millionChecksPerMonth > 1000) {
+      self.org.strChecksPerMonth = 'using ' + Math.ceil(millionChecksPerMonth / 1000) + ' Billion checks/mo';
+    } else if (millionChecksPerMonth > 0) {
+      self.org.strChecksPerMonth = 'using ' + millionChecksPerMonth + ' Million checks/mo';
+    } else {
+      self.org.strChecksPerMonth = 'not using any checks yet';
+    }
+
+    self.requiresUpgrade = self._requiresUpgrade();
+    self.currentlyTrial = self._currentlyTrial();
+    self.aboveFreeTier = self._aboveFreeTier();
```

Comment:
I'm thinking that on snapshots we don't actually want to show any of the upgrade/trial/free tier/checks-per-month stuff.  Thoughts @mattttt ?